content/rs/security/access-control/manage-users/login-lockout.md

```diff
@@ -19,89 +19,164 @@ Here, you learn how to configure the relevant settings.
+- [Cluster Manager UI](#configure-user-login-lockout)
 
-```sh
-rladmin tune cluster login_lockout_duration 3600
-```
+- [`rladmin tune cluster`]({{<relref "/rs/references/cli-utilities/rladmin/tune#tune-cluster">}}):
+
+    ```sh
+    rladmin tune cluster login_lockout_threshold <integer>
+    ```
+
+    For example, to set the lockout threshold to 10 failed login attempts, run:
+
+    ```sh
+    rladmin tune cluster login_lockout_threshold 10
+    ```
+
+    If you set the lockout threshold to 0, it turns off account lockout, and the cluster settings show `login_lockout_threshold: disabled`.
+
+    ```sh
+    rladmin tune cluster login_lockout_threshold 0
+    ```
+
+### Change time before login attempts reset
+
+To change the amount of time during which failed login attempts are counted, use one of the following methods:
+
+- [Cluster Manager UI](#configure-user-login-lockout)
+
+- [`rladmin tune cluster`]({{<relref "/rs/references/cli-utilities/rladmin/tune#tune-cluster">}}):
+
+    ```sh
+    rladmin tune cluster login_lockout_counter_reset_after <seconds>
+    ```
+
+    For example, to set the lockout reset to 1 hour, run:
+
+    ```sh
+    rladmin tune cluster login_lockout_counter_reset_after 3600
+    ```
 
-If you set the lockout duration to 0, then the account can be unlocked only when an administrator changes the account's password. In this case, the cluster settings show `login_lockout_duration: admin-release`.
+### Change login lockout duration
+
+To change the amount of time that the user account is locked after excessive failed login attempts, use one of the following methods:
+
+- [Cluster Manager UI](#configure-user-login-lockout)
+
+- [`rladmin tune cluster`]({{<relref "/rs/references/cli-utilities/rladmin/tune#tune-cluster">}}):
+
+    ```sh
+    rladmin tune cluster login_lockout_duration <login_lockout_duration>
+    ```
+
+    For example, to set the lockout duration to 1 hour, run:
+
+    ```sh
+    rladmin tune cluster login_lockout_duration 3600
+    ```
+
+    If you set the lockout duration to 0, then the account can be unlocked only when an administrator changes the account's password.
+
+    ```sh
+    rladmin tune cluster login_lockout_duration 0
+    ```
+
+    The cluster settings now show `login_lockout_duration: admin-release`.
 
 ### Unlock locked user accounts
 
 To unlock a user account or reset a user password with `rladmin`, run:
 
 ```sh
-rladmin cluster reset_password <user email>
+rladmin cluster reset_password <user_email>
 ```
 
 To unlock a user account or reset a user password with the REST API, use [`PUT /v1/users`]({{< relref "/rs/references/rest-api/requests/users#put-user" >}}):
 
 ```sh
-PUT https://[host][:port]/v1/users
-    '{"password": "<new_password>"}'
+PUT /v1/users
+{"password": "<new_password>"}
 ```
 
-## Session timeout
+### Turn off login lockout
+
+To turn off user login lockout and allow unlimited login attempts, use one of the following methods:
```

Comment:
This is worded really clearly. I like how you included the impact of changing the setting!